test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -571,12 +585,26 @@ class AccountListPage {
     console.log(
       `Verify the number of accounts in the account menu is: ${expectedNumberOfAccounts}`,
     );
-    await this.driver.wait(async () => {
-      const internalAccounts = await this.driver.findElements(
-        this.accountListItem,
-      );
-      return internalAccounts.length === expectedNumberOfAccounts;
-    }, 20000);
+
+    await this.driver.waitForSelector(this.accountListItem);
+    await this.driver.wait(
+      async () => {
+        await this.driver.delay(500);
```

Comment:
we don't need this line, we can just use the default polling frequency defined in `wait()`

Reply:
fixed